<?xml version="1.0"?>
<launch>

    <arg name="debug" default="false"/>

    <node pkg="hero_bridge" type="manipulation_bridge.py" name="manipulation_bridge" >
        <remap from="/hsrb/joint_states" to="joint_states" />
        <remap from="/hsrb/arm_trajectory_controller/follow_joint_trajectory/" to="body/joint_trajectory_action" />
        <remap from="/hsrb/arm_trajectory_controller/joints" to="arm_trajectory_controller/joints" />
        <remap from="/hsrb/head_trajectory_controller/follow_joint_trajectory/" to="head_trajectory_controller/follow_joint_trajectory" />
        <remap from="/hsrb/head_trajectory_controller/joints" to="head_trajectory_controller/joints" />
        <remap from="/hsrb/gripper_controller/follow_joint_trajectory/" to="gripper_controller/follow_joint_trajectory" />
        <remap from="/hsrb/gripper_controller/joints" to="gripper_controller/joints" />
        <remap from="/hsrb/omni_base_controller/follow_joint_trajectory/" to="omni_base_controller/follow_joint_trajectory" />
        <remap from="/hsrb/omni_base_controller/base_coordinates" to="omni_base_controller/base_coordinates" />
        <remap from="/hsrb/impedance_control/follow_joint_trajectory/" to="impedance_control/follow_joint_trajectory" />
        <remap from="/hsrb/impedance_control/joint_names" to="impedance_control/joint_names" />
        <remap from="/trajectory_filter/filter_trajectory_with_constraints" to="trajectory_filter/filter_trajectory_with_constraints" />
        <remap from="/plan_with_constraints" to="plan_with_constraints"/>
        <remap from="/plan_with_hand_goals" to="plan_with_hand_goals"/>
        <remap from="/plan_with_hand_line" to="plan_with_hand_line"/>
        <remap from="/plan_with_joint_goals" to="plan_with_joint_goals"/>
        <remap from="/filter_hsrb_trajectory" to="filter_hsrb_trajectory"/>
        <!-- collision world -->
        <remap from="/get_collision_environment" to="get_collision_environment"/>
        <remap from="/known_object" to="known_object"/>
        <remap from="/known_object_ids" to="known_object_ids"/>

        <remap from="/robot_description" to="robot_description"/>
    </node>
</launch>
